lib/commands/order/all_food_orders.rb

```diff
@@ -8,16 +8,16 @@ def applies_to?(request)
       "all food orders" == request
     end
 
-    def prepare(data) end
+    def prepare(data)
+      @crafter = User.profile(data[:user_id])
+    end
 
     def run
-      count_meal = Order.aggregate(
-        :lunch,
-        :all.count,
-        date: Days.from_monday_to_friday
-      )
+      counted_lunch = Order.placed_in(@crafter.office).each_with_object(Hash.new(0)) do |obj, hash|
+        hash[obj.lunch] += 1
```

Comment:
`@user`?

Reply:
and I think you can just write `{}` instead of `Hash.new(0)`

Reply:
`{}` does initialiser value at `nil` instead of `0`